testing/sdk_cmd.py

```diff
@@ -22,6 +23,7 @@ def fn():
         return fn()
 
 
+@retry(stop_max_attempt_number=5, stop_max_delay=60)
```

Comment:
The failures I saw had separate call chains as `run_cli` is also called outside of the `read_secret` function in the secrets tests.